src/plotSearch/components/plotSearchSections/application/ApplicationPreviewSection.js

```diff
@@ -171,6 +184,30 @@ class ApplicationPreviewSection extends PureComponent<Props> {
           large: 3,
         };
         break;
+      case 'hidden':
+        fieldSpecificComponents = <FormField
+          name={fakeFieldId}
+          fieldAttributes={{
+            type: 'hidden',
+            required: false,
+            read_only: true,
+            value: field.default_value,
+          }}
+          readOnlyValueRenderer={() => {
```

Comment:
I'd probably actually want to show the field label here like with other fields or otherwise indicate what the text is for, just for clarity.

Reply:
I see. Good point!